api/python/tests/integration/test_packages.py

```diff
@@ -69,6 +69,54 @@ def test_build(tmpdir):
         pkg = Package.load(fd)
         assert test_file.resolve().as_uri() == pkg['bar'].physical_keys[0]
 
+@patch('appdirs.user_data_dir', lambda x,y: os.path.join('test_appdir', x))
+def test_default_registry(tmpdir):
+    new_pkg = Package()
+
+    # Create a dummy file to add to the package.
+    test_file_name = 'bar'
+    with open(test_file_name, "w") as fd:
+        fd.write('test_file_content_string')
+        test_file = Path(fd.name)
```

Comment:
Btw, `Path` has a shortcut for writing files -`write_text(...)` and `write_binary(...)`. Much simpler than `with open(...)`.